language/borrow-graph/src/graph.rs

```diff
@@ -90,18 +90,6 @@ impl<Loc: Copy, Lbl: Clone + Ord> BorrowGraph<Loc, Lbl> {
         returned_edges
     }
 
-    /// Return the incoming edges into id
-    pub fn in_edges(&self, id: RefID) -> Vec<(Loc, RefID, Path<Lbl>, bool)> {
```

Comment:
Please revert

Reply:
A note: just last week we absolutely needed this function to work with the borrow graph. We changed representation and not using the borrow graph anymore, but it is an essential functionality for a data structure like this. This is a good example how difficult it is to decide whether to keep such code or not.